src/status_im/ui/screens/chat/views.cljs

```diff
@@ -285,8 +291,7 @@
         all-loaded? @(re-frame/subscribe [:chats/all-loaded? chat-id])]
     [react/view {:style (when platform/android? {:scaleY -1})}
      (if (or loading-messages? (not chat-id) (not all-loaded?))
-       [react/view {:height 324 :align-items :center :justify-content :center}
-        [react/activity-indicator {:animating true}]]
+       [messages-skeleton/messages-skeleton @messages-view-height]
```

Comment:
we need to implement this only for new design, so two ways , 1 - duplicate list-footer, or 2- use new-ui atom condition

Reply:
Duplicated list footer for better control.